src/style.css

```
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html * {
```

```suggestion
html {
```
don't use nesting, `*` might have only `box-sizing: border-box;`, everything else is forbidden

---

if I place  into html selector  { border: none;  outline: none; }  
as requested earlier,  than it will not work.

https://github.com/mate-academy/layout_search-bar-airbnb/pull/2555#discussion_r1049482815

---

Can I leave only font-family in html selector,
and move border and outline styles into .searchInput?

Like it was in the beginning:
https://github.com/mate-academy/layout_search-bar-airbnb/pull/2555#discussion_r1045297585

---

Nesting removed, border and outline also moved to input styles.